### PR TITLE
Practice app/quiz feature

### PR DESCRIPTION
**Pull request by hsuleerkul**
I have implemented backend, and frontend for quizzes. also added comments for swagger, have't tested yet. After merge I will test it. For unit tests I have implemented most of them.

### CODE REVIEW

**Comment by Goko6** [comment]
I detected a syntax error in quiz.route.js file. The review is still on progress
image
Adding a comma at the end of "authorization" will solve the error.

**Comment by hsuleerkul** [comment]
> I detected a syntax error in quiz.route.js file. The review is still on progress image Adding a comma at the end of "authorization" will solve the error.

Oh thanks, probably removed just before committing. Fixed it!

File: practice-app/server/src/routes/api.js [hunk 1]
```diff
@@ -1,16 +1,19 @@
 const express = require("express");
+
+const api = express.Router();
```
Comment (Goko6) [inline practice-app/server/src/routes/api.js:3]: The line: "const api = express.Router();" appears multiple times in the code. The upper one should be deleted

**Review by Goko6 — CHANGES_REQUESTED** [review]

File: practice-app/server/src/routes/api.js (hunk 1 above)
Comment (hsuleerkul) [reply practice-app/server/src/routes/api.js:3]: I changed it in [this commit](https://github.com/bounswe/bounswe2022group3/commit/f52dd420ca710d9cafa4e3a6137cff0fd60bc7c9) after merging my code with master. Thank you for the reminder!

**Review by Goko6 — APPROVED** [review]
The code looks good and I think that it is ready to merge

**Pull request merged**